### ISSUE TITLE
Hook up to XRayCI

### ISSUE THREAD

**Issue by kevgo** [Originate/git-town#384]
After we are on CircleCI, let's track how test times evolve on https://xrayci.herokuapp.com.

Blocked on #377

**Comment by allewun** [comment]
Cool! Would've been nice to have this before and after the test speedup commits.

**Comment by kevgo** [comment]
Yeah, indeed! Let's make it happen!

**Comment by kevgo** [comment]
This is done now.
